contrib/update-submodules.sh

```diff
@@ -8,4 +8,4 @@ WORKDIR=$(readlink -f "${WORKDIR}")
 "$WORKDIR/sparse-checkout/setup-sparse-checkout.sh"
```

Comment:
```suggestion
cd $"WORKDIR" || exit 1
"$WORKDIR/sparse-checkout/setup-sparse-checkout.sh"
```

It feels like this is why it failed recently for me. Because it works from the PWD of the parent process, and not from the git repo.

Reply:
Will fix.